gradle/versions.gradle

```diff
@@ -76,6 +76,7 @@ dependencyManagement {
 
     dependency 'org.awaitility:awaitility:4.0.1'
 
+    dependency 'org.bouncycastle:bcpkix-jdk15on:1.64'
```

Comment:
duplicated dependency?

Reply:
Where is it duplicated? The existing one is `bcprov`, the new one in `bcpkix`.

Reply:
Where is it duplicated? The existing one is `bcprov`, the new one in `bcpkix`.